swagger.yaml

```diff
@@ -19,6 +19,98 @@ schemes:
 basePath: '/v1'
 
 paths:
+  /repositories:
+    post:
+      summary: Create a new repository
+      operationId: createRepository
+      tags:
+        - repositories
+      parameters:
+        - in: body
+          name: newRepository
+          required: true
+          description: The repository to create
+          schema:
+            type: object
+            required:
+              - name
+            properties:
+              name:
+                type: string
+              description:
+                type: string
+              website:
+                type: string
```

Comment:
Isn't there a url type?

Reply:
Sadly there's not, but we check during validation with `govalidator.IsURL`